src/commands/spawn.h

```diff
@@ -25,8 +26,42 @@ class SpawnCommand : public ICommand
         transform.m[3].y = aimpos.y + 1.0f;
         transform.m[3].z = aimpos.z;
 
-        // NOTE(aaronlad): spawn flag 0x8000 will prevent auto despawn
-        jc::CSpawnSystem::instance().Spawn(arguments, transform, [](const jc::spawned_objects &objects, void *) {});
+        // figure out if the player wants to spawn more than one thing
+        if (arguments.find("*", 0) != std::string::npos) {
+            // identify the multiplier and turn it into something we can use, then create a string without it
+            // also identify the spacer to enable spawning of bigger things
+            // limiting the size of the substring limits max spawned entities, although the game seems to limit this itself
+            size_t multiplier_position = arguments.find("*", 0);
+            size_t spacer_position     = arguments.find("/", 0);
+            std::string truncated_args = arguments.substr(0, multiplier_position - 1);
+
+            int multiplier             = std::stoi(arguments.substr(multiplier_position + 1, 4)); 
```

Comment:
Just thinking if `atoi` would be better here (and line 42). `std::stoi` will throw an exception if the conversion fails, `atoi` will simply return 0, which I think would be OK here instead of doing exception handling.